### ISSUE TITLE
Imgur now forcing captchas on upload via URL method 

### ISSUE THREAD

**Issue by philoushka** [oscarfroberg/imguruploader#3]
![image](https://cloud.githubusercontent.com/assets/930731/7280784/017e87fe-e8da-11e4-8c8c-7af6e1e8d8a7.png)

Perhaps the solution is to move to the new Imgur API Version 3?

**Comment by oscarfroberg** [comment]
Yeah, I'd do that if I wasn't so lazy. However, the captcha only shows for some users some amount of the time.
